modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppStepContent.es.js

```diff
@@ -70,6 +74,29 @@ const EditAppStepContent = ({
 				{shortCutButton}
 			</ClayLayout.ContentRow>
 
+			<ClayLayout.ContentRow className="pl-4 pr-4">
+				{showInfoAlert && (
+					<ClayAlert
+						className="w-100"
+						displayType="info"
+						onClose={() => {
+							setShowInfoAlert(false);
+						}}
+						title={`${Liferay.Language.get('info')}:`}
+					>
+						{Liferay.Language.get(
+							'refresh-the-list-if-a-newly-created-workflow-is-not-displayed'
+						)}
+
+						<ClayLink
+							onClick={() => {
+								refetch();
+							}}
```

Comment:
@naraandrade, I think that this alert implementation is specific for the workflow step, what do you think about implement a generic alert receiving the alert component or the alert props inside the step content props?

Reply:
Great suggestion :) thank you